src/content/docs/en/reference/api-reference.mdx

```diff
@@ -778,10 +826,79 @@ const draftBlogPosts = await getCollection('blog', ({ data }) => {
 
 [See the `Content Collection` guide](/en/guides/content-collections/#querying-collections) for example usage.
 
+### `getEntry()`
+
+<p>
+  <Since v="2.5.0" />
+</p>
+
+**Types:**
+
+- `(collection: string, contentSlugOrDataId: string) => CollectionEntry<collection>`
+- `({ collection: string, id: string }) => CollectionEntry<collection>`
+- `({ collection: string, slug: string }) => CollectionEntry<collection>`
+
+`getEntry()` is a function that retrieves a single collection entry by collection name and either the entry `id` (for JSON or YAML collections) or entry `slug` (for Markdown or MDX collections).
+
+```astro
+---
+import { getEntry } from 'astro:content';
+
+// Get `src/content/blog/enterprise.md`
+const enterprisePost = await getEntry('blog', 'enterprise');
+
+// Get `src/content/captains/captain-kirk.yaml`
+const captainProfile = await getEntry('captains', 'captain-kirk');
+---
+```
+
+`getEntry()` can also be used to get referenced entries to access the `data`, `body`, or `render()` properties:
+
+```astro
+---
+import { getEntry } from 'astro:content';
+
+const enterprisePost = await getEntry('blog', 'enterprise');
+
+// Get the profile referenced by `data.captain`
+const captainProfile = await getEntry(enterprise.data.captain);
+---
+```
+
+See the `Content Collection` guide for examples of [querying](/en/guides/content-collections/#querying-collections) and [references](/en/guides/content-collections/#resolving-references).
+
+### `getEntries()`
+
+<p>
+  <Since v="2.5" />
+</p>
+
+**Types:**
+
+- `(Array<{ collection: string, id: string }>) => CollectionEntry<collection>`
+- `(Array<{ collection: string, slug: string }>) => CollectionEntry<collection>`
+
+`getEntries()` is a function that retrieves multiple collection entries with the same collection name. This is useful for resolving an array of [references](en/guides/content-collections/#defining-collection-references) to their associated `data`, `body`, and `render()` properties.
```

Comment:
```suggestion
`getEntries()` is a function that retrieves multiple collection entries from the same collection. This is useful for [returning an array of referenced entries](/en/guides/content-collections/#defining-collection-references) to access their associated `data`, `body`, and `render()` properties.
```
Fixed the link syntax. 

Also, "references" felt like a strange link text. Out of context, it doesn't mean very much, so it should be a slightly longer phrase, I think.

AND we don't really define/use "a reference, n." elsewhere -- we talk about the `reference()` utility. I'm seeing how "a referenced entry" feels as the noun that we use.

non-blocking nit: "retrieve/return for access to properties" feels a little more our existing style vs. "resolve to."